sdk/metricsadvisor/Azure.AI.MetricsAdvisor/src/Models/Credential/DatasourceCredential.cs

```diff
@@ -42,5 +43,22 @@ internal DatasourceCredential(string name)
         /// </summary>
         [CodeGenMember("DataSourceCredentialDescription")]
         public string Description { get; set; }
+
+        internal DataSourceCredentialPatch GetPatchModel()
```

Comment:
The service uses different definitions for Update (in this case, a `DataSourceCredentialPatch`), so we need to manually convert from what we expose to what the service specifies.

Reply:
Correct.